samples/csharp_dotnetcore/50.teams-messaging-extensions-search/TeamsAppManifest/manifest.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.schema.json",
   "manifestVersion": "1.5",
   "version": "1.0",
-  "id": "<<YOUR-MICROSOFT-APP-ID>>",
```

Comment:
Please change this back, since the readme directly references `<<YOUR-MICROSOFT-APP-ID>>`